cli/src/session.rs

```diff
@@ -141,7 +142,7 @@ impl Session {
             }
             // server keywords
             if !settings.no_auto_complete {
-                let rows = conn.query_iter(PROMPT_SQL).await;
+                let rows = conn.query_iter(PROMPT_SQL, ()).await;
```

Comment:
It's a little weird to pass an empty everytime, maybe we could add some new function like `query_iter_params`?

Reply:
Rust does not support optional arguments, if we add new functions there will be 

`query_iter_params` and `query_all_params` , `exec_params`, `query_row_params`, `query_iter_ext_params` ....

Reply:
> It's a little weird to pass an empty param everytime, maybe we could add some new function like `query_iter_params`?

If you don't want to pass, you can use `conn.inner().query_iter(PROMPT_SQL)`